Namespace testcase fails to pass

In a fresh checkout of `pydap` in a clean venv, the `namespace` testcase does not pass:
```
======================================================================
FAIL: Test the namespace.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/gvaillan/.virtualenvs/pydap/src/pydap/src/pydap/tests/test_pydap.py", line 20, in test_namespace
    pydap.__doc__, "Declare the namespace ``pydap`` here.")
AssertionError: None != 'Declare the namespace ``pydap`` here.'

----------------------------------------------------------------------
Ran 311 tests in 0.582s
```
Indeed, if I manually run:
```python
>>> import pydap
>>> print(pydap.__doc__)
None
```
I am wondering whether this test case is actually useful, beyond increasing the code coverage?

Did you use a *fresh* virtualenv?

Steps:
```
virtualenv --python=python3 ~/.virtualenvs/pydap
cd ~/.virtualenvs/pydap && source ./bin/activate
mkdir src && cd src
git clone https://github.com/pydap/pydap && cd pydap
pip install -e .[testing,functions,tests]
nosetests -s pydap.tests.test_pydap
```
Output:
```
F
======================================================================
FAIL: Test the namespace.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "<HOME>/.virtualenvs/pydap/src/pydap/src/pydap/tests/test_pydap.py", line 20, in test_namespace
    pydap.__doc__, "Declare the namespace ``pydap`` here.")
AssertionError: None != 'Declare the namespace ``pydap`` here.'

----------------------------------------------------------------------
Ran 1 test in 0.002s

FAILED (failures=1)
```

--- a/src/pydap/__init__.py
+++ b/src/pydap/__init__.py
@@ -1,2 +1,5 @@
-"""Declare the namespace ``pydap`` here."""
+'''
+Declare the namespace ``pydap`` here.
+'''
+
 __import__('pkg_resources').declare_namespace(__name__)

--- a/src/pydap/model.py
+++ b/src/pydap/model.py
@@ -504,8 +504,8 @@ class SequenceType(StructureType):
             out = SequenceType(self.name, self.data, self.attributes.copy())
             for name in key:
                 out[name] = copy.copy(StructureType.__getitem__(self, name))
-            # copy.copy() is necessary here because a view will be returned in the
-            # future:
+            # copy.copy() is necessary here because a view will be returned in
+            # the future:
             out.data = copy.copy(self.data[list(key)])
             return out
 

--- a/src/pydap/model.py
+++ b/src/pydap/model.py
@@ -504,7 +504,9 @@ class SequenceType(StructureType):
             out = SequenceType(self.name, self.data, self.attributes.copy())
             for name in key:
                 out[name] = copy.copy(StructureType.__getitem__(self, name))
-            out.data = self.data[list(key)]
+            # copy.copy() is necessary here because a view will be returned in the
+            # future:
+            out.data = copy.copy(self.data[list(key)])
             return out
 
         # Else return a new `SequenceType` with the data sliced.

--- a/src/pydap/tests/test_pydap.py
+++ b/src/pydap/tests/test_pydap.py
@@ -11,5 +11,3 @@ class TestNamespace(unittest.TestCase):
     def test_namespace(self):
         """Test the namespace."""
         self.assertEqual(pydap.__name__, "pydap")
-        self.assertEqual(
-            pydap.__doc__, "Declare the namespace ``pydap`` here.")

